src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +217,13 @@ func runClientProxy(cfg *config.Config) {
 	settings.Configure(cfg, version, buildDate)
 	proxiedsites.Configure(cfg.ProxiedSites)
 
+	httpClient, er := util.HTTPClient(cfg.CloudConfigCA, cfg.Addr)
```

Comment:
Yeah this is interesting. I *think* CloudConfigCA  will actually be blank at this point typically, which I *think* means it'll use the Go defaults. We definitely have a related issue on the FT side as well.

I guess we should actually pull out the Google Analytics CA here though huh?